emails/payin_bankwire_expired.spt

```diff
@@ -1,11 +1,11 @@
-{{ _("The bank wire to your Liberapay wallet has expired") }}
+{{ _("The initiated bank wire to your Liberapay wallet has expired") }}
 
 [---] text/html
 <p>{{ _(
     "On {date} you declared that you were going to transfer {money_amount} from your bank account to your Liberapay wallet, but the money never arrived.",
     date=exchange.timestamp.date(), money_amount=LegacyMoney(exchange.amount + exchange.fee)
 ) }}</p>
 
-<p>{{ _("If you forgot to do it you can initiate a new payment:") }}</p>
+<p>{{ _("If you forgot to do it you can initiate a new bank wire:") }}</p>
```

Comment:
If this shouldnt be "bank wire", then it follows that the money shouldn't come from "bank account". Either this, or scrapping "from your bank account" and going with "payment" are valid options.

Reply:
Add a comma after the dependent clause (see #959)

Reply:
Will do